src/lib/layers/editable-geojson-layer.js

```diff
@@ -28,6 +29,8 @@ const defaultProps = {
   // Edit and interaction events
   onEdit: () => {},
 
+  hintPointMaxDistance: 0,
+  showHintPoint: true,
```

Comment:
I don't think this needs to be configurable. Just show it if and only if `mode=modify`.

Reply:
Do you mind if we leave it for now? Our app's design spec doesn't call for a hint, despite it being a great feature.